libs/cases/frontend/view/src/lib/components/create-case/create-case.component.html

```diff
@@ -1,18 +1,18 @@
 <ix-content>
-	<ix-content-header slot="header" header-title="Create order"> </ix-content-header>
+	<ix-content-header slot="header" header-title="Create case"> </ix-content-header>
```

Comment:
the upper/lowercase should be consistent. Sometimes its says "case" and sometimes "Case" so I would suggest always writing it big (in the detail.page.ts it says "New Case" so uppercase)